sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 - Add support for session-level sanitization using the `Recorder.addSessionSanitizers` static method. [#21533](https://github.com/Azure/azure-sdk-for-js/pull/21533)
 - Added logging to help with debugging tests. [#21641](https://github.com/Azure/azure-sdk-for-js/pull/21641)
+- Allow mapping the test-proxy tool to ports other than just 5000(for HTTP) using the environment variable `TEST_PROXY_HTTP_PORT`.
+  - If `TEST_PROXY_HTTP_PORT` is undefined, we'll try for 5000 as usual.
+  - For browsers, this variable has to be added as part of the environment variables listed under `envPreprocessor` array in `karma.conf.js` so that the recorder knows the port to hit.
```

Comment:
Do we want to update the existing karma.conf.js files from our migrated packages?

We also should update the README documenting this new flag, in the other PR.

Reply:
> Do we want to update the existing karma.conf.js files from our migrated packages?

Let's do it lazily? If someone actually needs it, they can add it to their package.
And also, we have a plan to unify the karma configs, we could add it as a default.

> We also should update the README documenting this new flag, in the other PR.

That's my plan, yes.

Reply:
Yup, sounds good.